### PR TITLE
Improve the runserver script

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: runserver.py [hunk 1]
```diff
@@ -1,9 +1,51 @@
...
+    help='Hostname to listen on. When set to 0.0.0.0 the server is available '
+         'externally. Defaults to 127.0.0.1 making the it only visable on '
+         'localhost')
+
+args = parser.parse_args()
+
+if args.config:
+    config = args.config
+    if not config.startswith('/'):
+        here = os.path.join(os.path.dirname(os.path.abspath(__file__)))
```
Comment (sayanchowdhury) [inline runserver.py]: maybe replace this with `os.getcwd()`?

Comment (pypingou) [inline runserver.py]: The current working directory could be different from the directory where this file is located :)

**Comment by sayanchowdhury** [comment]
:+1:  for me

File: runserver.py (hunk 1 above)
Comment (pypingou) [inline runserver.py]: but the current directory might be more correct, you're right, I'll adjust

**Comment by sayanchowdhury** [comment]
:+1:

**Pull request merged**